examples/3d/parenting.rs

```diff
@@ -34,12 +37,18 @@ fn setup(
         ..default()
     });
 
+    let capsule_handle = meshes.add(Mesh::from(shape::Capsule::default()));
+    let capsule_material_handle = materials.add(StandardMaterial {
+        base_color: Color::rgb(0.7, 0.8, 0.6),
```

Comment:
I think color constants are generally more legible in these examples.

Reply:
Yes. I tried to keep the consistency with how the example was before, but I can change these for sure.

Reply:
Yep, may as well tidy up a bit while we're here :)

Reply:
Actually, I'm going to use a single material for all meshes to simplify the example and focus it more on the parenting. I'll keep the capsule though to keep comments clear and not have multiple "child cubes".